device_calendar/pubspec.yaml

```diff
@@ -7,11 +7,15 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.1.8
+  pedantic: ^1.8.0+1
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
 
+dependency_overrides:
```

Comment:
Can we update the main dependencies instead of using dependency_overrides?

Reply:
Sorry I forgot to mention about that. I have to use override or use an old version for now: https://github.com/dart-lang/pedantic/issues/46

Reply:
Also added this as a comment in the file for a reference